### ISSUE TITLE
Failed to validate prompt for output

### ISSUE THREAD

**Issue by Blackjack356** [kijai/ComfyUI-ADMotionDirector#20]
I don't know if its because the admotiondirector isnt updated but i have a new problem now there it is:

got prompt
Failed to validate prompt for output 26:
* ADMD_TrainLora 24:
  - Return type mismatch between linked nodes: trigger_input, ADMDPIPELINE != VHS_FILENAMES
  - Required input is missing: admd_pipeline
Output will be ignored
Failed to validate prompt for output 14:
* ADMD_TrainLora 12:
  - Return type mismatch between linked nodes: trigger_input, ADMDPIPELINE != VHS_FILENAMES
  - Required input is missing: admd_pipeline
Output will be ignored
Failed to validate prompt for output 64:
* ADMD_TrainLora 32:
  - Return type mismatch between linked nodes: trigger_input, ADMDPIPELINE != VHS_FILENAMES
  - Required input is missing: admd_pipeline
Output will be ignored
Failed to validate prompt for output 34:
Output will be ignored
Failed to validate prompt for output 11:
* ADMD_TrainLora 8:
  - Return type mismatch between linked nodes: trigger_input, ADMDPIPELINE != VHS_FILENAMES
  - Required input is missing: admd_pipeline
Output will be ignored
Failed to validate prompt for output 30:
* ADMD_TrainLora 28:
  - Return type mismatch between linked nodes: trigger_input, ADMDPIPELINE != VHS_FILENAMES
  - Required input is missing: admd_pipeline
Output will be ignored
WARNING: ADMD_CheckpointLoader.IS_CHANGED() got an unexpected keyword argument 'ckpt_name'
Some weights of the model checkpoint were not used when initializing CLIPTextModel:
 ['text_model.embeddings.position_ids']
using DDIMScheduler for training
load motion module from G:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-AnimateDiff-Evolved\models\v3_sd15_mm.ckpt

**Comment by Blackjack356** [comment]
G:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-ADMotionDirector\animatediff\utils\util.py:117: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  motion_module_state_dict = torch.load(motion_model, map_location="cpu")
load domain lora from G:\ComfyUI_windows_portable\ComfyUI\models\loras\v3_sd15_adapter.ckpt
G:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-ADMotionDirector\animatediff\utils\util.py:170: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  domain_lora_state_dict = torch.load(adapter_lora_path, map_location="cpu")
Processing interrupted
Prompt executed in 24.88 seconds

**Comment by Blackjack356** [comment]
so i tried the suggested pytorch 2.2.0 cu1.2.1 since i have the newest  2.4.1cu124 but its not working either

**Comment by Blackjack356** [comment]
Is this repo still alive? Its been 3 week without  response. I still have the same issue! And i don't even know what causing this! Probably an dependencies maybe ?

**Comment by chaseforks** [comment]
I've been getting the same error as well. It looks as though the input on the ADMD TrainLora Node have chaged but tried updating the repo, uninstalling reinstalling and can't seem to get it going. Frustrating because it was working fine just a few weeks ago

**Comment by kijai** [comment]
> I've been getting the same error as well. It looks as though the input on the ADMD TrainLora Node have chaged but tried updating the repo, uninstalling reinstalling and can't seem to get it going. Frustrating because it was working fine just a few weeks ago

Something about the new UI changes in Comfy changed and the old method I was using to force the model to reload on every run, as otherwise training won't start, never figured better workaround.

Seems to run now, didn't test more than that though. Added prodigy optimizer too if anyone wants to test it.

**Comment by chaseforks** [comment]
I got it to work! First, moved the pipeline connection from 'trigger input' to 'admd pipeline' then right clicked 'recreate node' which makes the steps input appear. Then you can relink the steps to that input and the workflow runs

**Comment by kijai** [comment]
> I got it to work! First, moved the pipeline connection from 'trigger input' to 'admd pipeline' then right clicked 'recreate node' which makes the steps input appear. Then you can relink the steps to that input and the workflow runs

Yeah that was another thing.. I hadn't updated the example workflows for the input order change, I did do that with the update now though.

**Comment by Blackjack356** [comment]
Thanks alot!
